streaming/src/test/scala/org/apache/spark/streaming/ReceivedBlockHandlerSuite.scala

```diff
@@ -57,10 +56,12 @@ class ReceivedBlockHandlerSuite
   val serializer = new KryoSerializer(conf)
   val manualClock = new ManualClock
   val blockManagerSize = 10000000
+  val blockManagerBuffer = new ArrayBuffer[BlockManager]()
```

Comment:
This seems like a huge anti-pattern to me! Unless you're testing the interaction of multiple block managers, creating a a buffer to manage cleanup seems to suggest bad test design to me.